# Request
int16 torque
---
# Response
bool success
